JS에서 TS로 변환 완료

jiwoo0629 opened the pull request:
index.html : 달력(datepicker)용 script 소스 추가
Router.tsx : 홈페이지 연결하는 router 추가
Calendar.tsx : 달력 용 컴포넌트
EachData.tsx : MainPage에서 각 데이터 하나에 해당하는 컴포넌트
MainPage_Sensor.tsx : MainPage에서 센서 한 줄에 해당하는 컴포넌트 SelectData.tsx : 공기질 데이터 선택하는 컴포넌트
SelectSensor.tsx : 센서 선택하는 컴포넌트
UpperPage.tsx : 홈페이지 상단의 로고 및 버튼들 담고 있는 컴포넌트
images 폴더 : 로고 및 버튼 디자인
ExportPage.tsx : 엑셀 export 화면
LoginPage.tsx : 최초 로그인 화면
MainPage.tsx : 로그인 직후 메인화면
TimeGraph.tsx : 시계열 데이터 출력 화면
UserInfo.tsx : 내 프로필 화면
Div.css : 각 div 및 컴포넌트들의 css를 담고있는 파일, 추후에 페이지 별 css 나눠서 정리하는 것도 고려중

아직 styled-component는 적용하지 않은 상태임.

jiwoo0629 committed:
index.html : 달력(datepicker)용 script 소스 추가
Router.tsx : 홈페이지 연결하는 router 추가
Calendar.tsx : 달력 용 컴포넌트
EachData.tsx : MainPage에서 각 데이터 하나에 해당하는 컴포넌트
MainPage_Sensor.tsx : MainPage에서 센서 한 줄에 해당하는 컴포넌트
SelectData.tsx : 공기질 데이터 선택하는 컴포넌트
SelectSensor.tsx : 센서 선택하는 컴포넌트
UpperPage.tsx : 홈페이지 상단의 로고 및 버튼들 담고 있는 컴포넌트
images 폴더 : 로고 및 버튼 디자인
ExportPage.tsx : 엑셀 export 화면
LoginPage.tsx : 최초 로그인 화면
MainPage.tsx : 로그인 직후 메인화면
TimeGraph.tsx : 시계열 데이터 출력 화면
UserInfo.tsx : 내 프로필 화면
Div.css : 각 div 및 컴포넌트들의 css를 담고있는 파일, 추후에 페이지 별 css 나눠서 정리하는 것도 고려중

아직 styled-component는 적용하지 않은 상태임.

ghost commented:
1. 처음 시작 base url을 아래와 같이 변경 해야함. 
![image](https://user-images.githubusercontent.com/55936770/192001049-60e8fe17-57d4-4456-8325-d98d5a7d4cc9.png)
2. 이미지 파일은 assets 폴더안에 image폴더에 저장하는 것이 좋을듯.
3. 나머지는 추후 회의를 통해 개선하면 될듯!

jiwoo0629 commented:
확인했습니다!!

jiwoo0629 committed:
images 폴더 assets 디렉토리 내부로 이동
UpperPage.tsx, LoginPage.tsx : images 폴더 import 변경
Router.tsx : URL basename 변경
(시작 주소가 localhost:3000/Front-End-TS)로 나와서 로그인 화면 path를 임의로 설정해놓음
나중에 회의 거쳐서 수정)

jiwoo0629 committed:
Router.tsx : basename 변경
UpperPage.tsx, LoginPage.tsx : a 태그에서 Link 태그로 변경
